File: public/code_samples/javascript/verify.js
const raw = ''

const requestOptions = {
  method: 'POST',
  body: raw,
  redirect: 'follow'
}

fetch('http://localhost:4000/api/v1/auth/verify', requestOptions)
  .then(response => response.text())
  .then(result => console.log(result))
  .catch(error => console.log('error', error))

File: public/code_samples/dart/verify.dart
var request = http.Request('POST', Uri.parse('http://localhost:4000/api/v1/auth/verify'));
request.body = '''''';

http.StreamedResponse response = await request.send();

if (response.statusCode == 200) {
  print(await response.stream.bytesToString());
}
else {
  print(response.reasonPhrase);
}
